api/client/client.go

```diff
@@ -3067,6 +3068,27 @@ func (c *Client) DeleteAllDatabaseServices(ctx context.Context) error {
 	return trace.Wrap(err)
 }
 
+// GetDatabaseObjectsImportRules retrieves all database object import rules.
+func (c *Client) GetDatabaseObjectsImportRules(ctx context.Context) ([]*dbobjectimportrulev1.DatabaseObjectImportRule, error) {
```

Comment:
nit:
```suggestion
func (c *Client) GetDatabaseObjectImportRules(ctx context.Context) ([]*dbobjectimportrulev1.DatabaseObjectImportRule, error) {
```
Should `DatabaseObjects` be plural?

Reply:
Don't we use `DatabaseObjectImportRule` or like `ListDatabaseObjectImportRules` elsewhere?

Reply:
I agree with @greedy52 on this one. I'm not a native speaker, but I find it more idiomatic to name a list of object an "object list". Like grocery list, guest list. A pack of wolves is a wolf pack, not a wolves pack. Here's some useful discussion: https://www.proz.com/forum/linguistics/245800-noun_list_is_noun_singular_or_plural.html

Reply:
Done.